ufo2fontir/src/source.rs

```diff
@@ -428,7 +428,24 @@ impl Work for GlyphIrWork {
         let gid = static_metadata
             .glyph_id(&self.glyph_name)
             .ok_or_else(|| WorkError::NoGlyphIdForName(self.glyph_name.clone()))?;
-        let glyph_ir = to_ir_glyph(&self.glyph_name, &self.glif_files)
+
+        // Migrate glif_files into internal coordinates
+        let axes: HashMap<_, _> = static_metadata.axes.iter().map(|a| (&a.name, a)).collect();
+        let mut glif_files = HashMap::new();
+        for (path, design_locations) in self.glif_files.iter() {
+            let internal_locations: Vec<InternalLocation> = design_locations
+                .iter()
+                .map(|design_location| {
+                    design_location
+                        .iter()
+                        .map(|(an, dl)| (an.clone(), dl.to_user(&axes.get(an).unwrap().converter)))
```

Comment:
if this is a Vec<InternalLocation>, why does this do `to_user`?

Reply:
Because I borked the typing of internal locations in #56 :( Fixing.